apps/docs/public/manifest.webmanifest

```diff
@@ -1,18 +1,17 @@
 {
   "name": "Puck",
-  "short_name": "Pck",
+  "short_name": "Puck",
+  "start_url": "/",
   "icons": [
     {
       "src": "/maskable_icon_x192.png",
       "sizes": "192x192",
-      "type": "image/png",
-      "purpose": "maskable"
+      "type": "image/png"
```

Comment:
What's the reason for removing the `purpose` field?

https://web.dev/articles/maskable-icon

Reply:
"any" seemed superfluous